internal/internal_task_handlers.go

```diff
@@ -1474,6 +1475,7 @@ func (wth *workflowTaskHandlerImpl) completeWorkflow(
 			Header:              contErr.Header,
 			Memo:                workflowContext.workflowInfo.Memo,
 			SearchAttributes:    workflowContext.workflowInfo.SearchAttributes,
+			RetryPolicy:         convertToPBRetryPolicy(workflowContext.workflowInfo.RetryPolicy),
```

Comment:
PB = peanut butter

Reply:
I leveraged this pun to name by Kotlin proto generator [pb and k](https://github.com/cretz/pb-and-k)